learn/variables_and_types.md

```diff
@@ -0,0 +1,200 @@
+## Variable Definition
+
+What is a variable in Python? It is a named location in the computer's memory that stores a value. It is like a container that can hold different types of data, such as numbers, strings or booleans.
+
+To create a variable in Python, you need to give it a name and assign a value to it using the assignment operator `=`.
+
+Here's an example of a variable declaring in Python:
+
+```python
+# create a variable called "my_variable" and assign the value 10 to it
+my_variable = 10
+
+# print the value of "my_variable"
+print(my_variable) # Output: 10
+```
+
+In this example we initialised the variable - created a variable named `my_variable` and assigned it the value `10`. We then printed the value of `my_variable` using the `print()` function. The output of the program was `10`, which is the value stored in the variable `my_variable`.
+
+## Variable Declaration
+
+In Python, you don't need to explicitly declare a variable before using it. You can create a variable simply by assigning a value to it. Here's an example:
+
+```python
```

Comment:
This code example is very similar to the above one. I guess we can leave only one in the text.